drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-backend/src/main/java/org/drools/workbench/screens/workitems/backend/server/WorkbenchConfigurationHelper.java

```diff
@@ -61,11 +61,22 @@ public ConfigGroup getWorkItemElementDefinitions() {
                                                                          "  ]\n" +
                                                                          "]" ) );
         group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_PARAMETER,
-                                                                 "\"MyParam|\" : new StringDataType()" ) );
+                                                                 "\"MyParam\" : new StringDataType()" ) );
         group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_RESULT,
-                                                                 "\"Result|\" : new ObjectDataType()" ) );
+                                                                 "\"Result\" : new ObjectDataType()" ) );
         group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_DISPLAY_NAME,
-                                                                 "\"displayName\" : \"My Task|\"" ) );
+                                                                 "\"displayName\" : \"My Task\"" ) );
+
+        group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_PARAMETER_VALUES,
+                "\"parameterValues\" : [\n" +
+                        "   \"MyParam\" : \"A,B,C\",   \n" +
```

Comment:
Definition shortcut have parameters names:
```javascript
    "parameters" : [ 
        "MyFirstParam" : new StringDataType(), 
        "MySecondParam" : new StringDataType(), 
        "MyThirdParam" : new ObjectDataType() 
    ]
```
I guess we can use "MyFirstParam" as default name here